src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      class="searchForm" 
+      method="get" 
+      data-qa="big"
+    >
+      <input 
+        class="searchField searchField--big" 
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        name="search"
+        data-qa="keypress"
+      >
+      <div class="iconSearch iconBig">
+      </div>
```

Comment:
```suggestion
      <div class="iconSearch iconBig"></div>
```

You can write in a single string for such cases.